common-util/src/main/java/org/sunbird/common/responsecode/ResponseCode.java

```diff
@@ -34,6 +34,9 @@
   errorInvalidConfigParamValue(
       ResponseMessage.Key.ERROR_INVALID_CONFIG_PARAM_VALUE,
       ResponseMessage.Message.ERROR_INVALID_CONFIG_PARAM_VALUE),
+  errorBadgeAssociationNotFound(
```

Comment:
add ids to the error message - so that user can know for which badges are not active